### PR TITLE
added a few dao methods

### PR DESCRIPTION
**Pull request by VivianLi98**
Added a few more methods in Launch and User dao. May need to add more in the future, but this seems good enough for now.

### CODE REVIEW

**Review by chrsbell — COMMENTED** [review]
Think it looks good!

**Review by joelc1225 — CHANGES_REQUESTED** [review]
looks good. just fix the conflict and i'll merge

**Comment by VivianLi98** [comment]
Fixed merge conflict

**Review by joelc1225 — APPROVED** [review]

File: app/src/main/java/com/example/voyagerx/MainActivity.kt [hunk 1]
```diff
@@ -39,5 +39,7 @@ class MainActivity : AppCompatActivity() {
             replace(R.id.frame, desiredFragment)
             commit()
         }
+
+        setContentView(R.layout.activity_main)
```
Comment (brownyte) [inline app/src/main/java/com/example/voyagerx/MainActivity.kt:43]: I think this line is preventing the view from updating when the bottom nav buttons are pressed.

Comment (joelc1225) [reply app/src/main/java/com/example/voyagerx/MainActivity.kt:43]: @VivianLi98 why do we need to call set content here again?

@brownyte , does it work when commenting out this line?

Comment (brownyte) [reply app/src/main/java/com/example/voyagerx/MainActivity.kt:43]: I was able to get the bottom navigation buttons to start working again once I removed that line in the branch I have been working in

**Pull request merged**